### ISSUE TITLE
Different Loss Functions

### ISSUE THREAD

**Issue by TobiArndt** [baal-org/baal#278]
Hi,
I really like your work!
I have got a question with respect to the loss functions: Is there a chance to use loss functions from the unsupervised regime? Like Student-Teacher Losses, where the final loss is the difference between the intermediate layer between a frozen teacher and a trainanble student network?
This could be important for filtering anomalies from a dataset, so that only the non-anomaly images can be used for training the anomaly detector.
Thank in advanced!

**Comment by Dref360** [comment]
Hello, sorry for the delay.

Do you mean that you take the loss as a measure of uncertainty?
